torchvision/utils.py

```diff
@@ -143,6 +143,7 @@ def draw_bounding_boxes(
     boxes: torch.Tensor,
     labels: Optional[List[str]] = None,
     colors: Optional[List[Union[str, Tuple[int, int, int]]]] = None,
+    fill: Optional[List[Union[str, Tuple[int, int, int, int]]]] = None,
```

Comment:
I'm on the fence on this one while trying to balance flexibility and easy-of-use in this API. 

I think that having this as a list of colors would require people to provide too many parameters and pick too many colors. If you recall the palette size was one of the considerations on the first PR around this util, so adding the requirement to define even more colors might lead us to problems faster.

I think it's probably better to turn this into a boolean and fill the box with the same color as defined rather than supporting different colors for border and fill. Thoughts?

Reply:
Yes correct, color palette is an issue which rises quickly. 

Lets keep to boolean

Reply:
For semi transparent fill, the fill is `Tuple[int, int, int, int]` and not `Tuple[int, int, int]`. So what defaulot value should we keep for alpha channel.